### DISCUSSION TITLE
Black Screen when nootedred enabled

### DISCUSSION THREAD

**Discussion by yonkleto** [ChefKissInc/NootedRed#263] [General]
Hi all,
I successfully installed Sonoma on my HP Probook 445 G9 laptop with AMD 5825U. But when I tried to boot into system is black screen after verbose. When I disabled Nootedred, it could enter the system but with only vram of 7 MB. SMBIOS model is MacBookPro16,3, VRAM is 512MB (The best BIOS could set)
Any ideas?

**Comment by VisualEhrmanntraut** [comment]
https://chefkissinc.github.io/nred ?
